x/cash/controller.go

```diff
@@ -6,13 +6,18 @@ import (
 	"github.com/iov-one/weave/x"
 )
 
+// CoinsMover is an interface for moving coins between accounts.
+type CoinMover interface {
+	// MoveCoins removes funds from the source account and adds them to the
+	// destination account. This operation is atomic.
```

Comment:
You had some questions on what amount can be (is zero allowed as noop? negative is an error right?).
Maybe you want to add that here to the interface spec

Reply:
Added in https://github.com/iov-one/weave/pull/359/commits/a2a085cdb0612f5faa2f416116aef4af16d87519